components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4147,6 +4153,62 @@ public static AuthenticatedUser createAuthenticatedUser(String username, String
         return authenticatedUser;
     }
 
+    /**
+     * Creates an instance of AuthenticatedUser{@link AuthenticatedUser} for the given parameters.
+     *
+     * @param username        username of the user
+     * @param userStoreDomain user store domain
+     * @param tenantDomain    tenent domain
+     * @param idpName         idp name
```

Comment:
missing fullstops.
and make the first letter capital in the comment

Reply:
Fix other places as well

Reply:
Fixed by - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2504/commits/13de5fc1f8327d2e23a21a5655193cb857635a69